api/src/pcapi/scripts/beneficiary/import_test_users.py

```diff
@@ -202,8 +202,13 @@ def _add_or_update_user_from_row(row: dict, update_if_exists: bool) -> User | No
     db.session.add(user)
     db.session.commit()
 
+    mail_anonymized = anonymize_email_from_user(user=user)
```

Comment:
```suggestion
    anonymized_mail = anonymize_email_from_user(user=user)
```

Reply:
Je dirais même plus, anonymized_**e**mail, par cohérence avec le nommage partout dans le code 
```suggestion
    anonymized_email = anonymize_email_from_user(user=user)
```